conda/recipes/librmm/meta.yaml

```diff
@@ -35,7 +35,7 @@ requirements:
     - cudatoolkit {{ cuda_version }}.*
   run:
     - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}
-    - spdlog=1.7.0
+    - spdlog>=1.8.2,<2.0.0a0
```

Comment:
Why is this 1.8.2 whereas everywhere else is 1.8.5?

Reply:
Ah missed that one. Fixed.